plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergGlueCatalogConnectorSmokeTest.java

```diff
@@ -121,14 +121,6 @@ public void testShowCreateTable()
                         schemaPath()));
     }
 
-    @Test
-    @Override
-    public void testMaterializedView()
```

Comment:
Smoke tests are testing only cases around CREATE, SELECT, SHOW CREATE and DROP.
We should be testing CREATE OR REPLACE, listing of MVs and RENAME as well with Glue. Is that already covered by other tests ?

Reply:
I didn't add any. Do you know if there's coverage in BaseConnectorTest? Yuya has a PR running that against Glue https://github.com/trinodb/trino/pull/11746

Reply:
For now I converted `TestIcebergMaterializedViews` to `BaseIcebergMaterializedViewTest` and added both a Hive and Glue backed implementation.